Modelplace_Cloud_API/api_requests.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import json
+
+
+def login_request(email: str, password:str) -> json:
```

Comment:
the file is named api-**requests** so you don't need to add `_requests` to each funcion